resources/home/dnanexus/utils/util_functions.py

```
@@ -1,22 +1,124 @@
 """General utility functions"""
 
+import os
+from typing import Tuple, Union
 
-def get_excluded_intervals(gcnv_output_dict):
+import dxpy
+
+
+def add_session_file_ids_to_job_output(all_sample_outputs, job_output) -> dict:
```

Ruff (0.8.0)

Missing type annotation for function argument `all_sample_outputs`

(ANN001)